### PR TITLE
add simplemachines.org

### PR DESCRIPTION
**Pull request by redranamber**
New rule from #9696

### CODE REVIEW

File: simplemachines.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<ruleset name="Simple Machines">
+	<target host="simplemachines.org"/>
+	<target host="www.simplemachines.org"/>
+	<target host="adsystem.simplemachines.org"/>
```
Comment (J0WI) [inline simplemachines.xml]: blogs.simplemachines.org

File: simplemachines.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<ruleset name="Simple Machines">
+	<target host="simplemachines.org"/>
+	<target host="www.simplemachines.org"/>
+	<target host="adsystem.simplemachines.org"/>
+	<target host="custom.simplemachines.org"/>
+	<target host="dev.simplemachines.org"/>
+	<target host="download.simplemachines.org"/>
+	<target host="wiki.simplemachines.org"/>
+	<target host="support.simplemachines.org"/>
+	<target host="media.simplemachinesweb.com"/>
```
Comment (J0WI) [inline simplemachines.xml]: mods.simplemachines.org

File: simplemachines.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	<target host="download.simplemachines.org"/>
+	<target host="wiki.simplemachines.org"/>
+	<target host="support.simplemachines.org"/>
+	<target host="media.simplemachinesweb.com"/>
+	<test url="http://custom.simplemachines.org/"/>
+	<test url="http://dev.simplemachines.org/"/>
+	<test url="http://download.simplemachines.org/"/>
+	<test url="http://dev.simplemachines.org/"/>
+	<test url="http://wiki.simplemachines.org/"/>
+	<test url="http://support.simplemachines.org/"/>
```
Comment (J0WI) [inline simplemachines.xml]: You don't need to add test urls as long as they are the same as the listed targets.

**Comment by redranamber** [comment]
Done

**Comment by J0WI** [comment]
Thanks!

**Comment by cschanaj** [comment]
@J0WI This is merged in the wrong folder #9851

**Comment by Hainish** [comment]
I removed `simplemachines.xml` from the git root in f0a60f07c96640f247cac859227b8dfda1403099, which should not interfere with #9973 as that PR simply adds a new file, rather than moving the previous file.

**Pull request merged**